Test/TestProject/TestProject.csproj

```diff
@@ -11,16 +11,20 @@
 
     <!--
         Normally a project just has to reference this package like this:
-        
+
         <PackageReference Include="dafny.msbuild" Version="*.*.*" />
-        
+
         But to test the functionality without packaging up the nupkg itself,
         we import the properties and target directly in the same way nuget
         will automatically instead.
     -->
     <Import Project="../../Source/dafny.msbuild/build/dafny.msbuild.props" />
     <Import Project="../../Source/dafny.msbuild/build/dafny.msbuild.targets" />
 
+    <PropertyGroup>
+        <DafnyVersion>2.3.0.10506</DafnyVersion>
```

Comment:
Hopefully the fresh CI run will catch this too, but this will need to be updated.

It might be better to use the `setup-dafny-action` GH action to fetch a specific version of Dafny instead of the latest tip of master, so this won't break whenever new versions are released.

Reply:
Ideally we could have tests in CI for the other two options as well.

Reply:
For now, I've just updated this to use tag v3.5.0.